web/client/themes/default/less/widget.less

```diff
@@ -316,3 +316,20 @@
 
 }
 
+.ms-widget-empty-message {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.ms-widget-empty-text-message {
```

Comment:
I would like to avoid duplicate this class only for the position property so I propose this approach:

```css
.ms-widget-empty-message {
    position: absolute;
    /* we ensure the empty table message is centered and cover the header with top at 0 and filled background */
    top: 0; 
    background-color: @ms2-color-background;
    width: 100%;
    height: 100%;
    display: flex;
    align-items: center;
    justify-content: center;
}
/* needed to compute the correct position of empty messages */
.mapstore-widget-card .ms2-border-layout-body {
    position: relative;
}
```

and remove the .ms-widget-empty-text-message class